kiwi/system/setup.py

```diff
@@ -87,6 +88,48 @@ def __init__(self, xml_state: XMLState, root_dir: str):
         self._preferences_lookup()
         self._oemconfig_lookup()
 
+    def setup_registry_import(self) -> None:
+        """
+        Fetch container(s) and activate systemd unit to load
+        fetched containers during boot
```

Comment:
s/fetched containers/containers from local tarball/

Reply:
fixed, I use oci-archive instead of local tarball